env-config/example.env

```diff
@@ -13,6 +13,7 @@ REACT_APP_SPRINGBOOT_BACKEND=
 REACT_APP_ASPNETCORE_BACKEND=
 REACT_APP_LARAVEL_BACKEND=
 REACT_APP_RUBY_BACKEND=
+REACT_APP_RAILS_BACKEND=
```

Comment:
This needs to be consistent with `ruby-on-rails` -> `RUBYONRAILS` for deploy.sh local port rewiring to work
https://github.com/sentry-demos/application-monitoring/blob/262aa3ec3b7f06605ce202acd3e611f7dd13ffa2/deploy.sh#L144

Reply:
@ndmanvar this should be `REACT_APP_RUBYONRAILS_BACKEND=` because the script automatically derives it from `ruby-on-rails` by removing dashes and capitalizing, kind of like with the variables prefixed with `RUBYONRAILS_`

Reply:
done. made the changes in `-config` 
https://github.com/sentry-demos/application-monitoring-config/pull/1/commits/4b6dfc4977b71b56f316a290efb878b52630be21